app/front/scripts/services/data-package-api/index.js

```diff
@@ -15,31 +16,50 @@ function loadConfig(settingsUrl) {
     module.exports.apiConfig = config.api;
     module.exports.searchConfig = config.search;
     module.exports.dataMineConfig = config.dataMine;
+    module.exports.osExplorerUrl = config.osExplorerUrl;
     return config;
   });
 }
 
-function getDataPackages(authToken) {
+function getDataPackages(authToken, packageId, userId) {
   return loadConfig().then(function() {
     var searchConfig = module.exports.searchConfig;
     var url = searchConfig.url + '?size=10000';
     if (authToken) {
       url += '&jwt=' + encodeURIComponent(authToken);
     }
+    if (packageId) {
```

Comment:
That's the thing - if you specify an id you will always get just one result.
If you want all packages from the user, you should specify just the owner and not the id.